dublincore

```diff
@@ -0,0 +1 @@
+Subproject commit 8397deb280d09cbb8e473409198755f9d73156eb
```

Comment:
@robline Just a FYI - dublincore submodule has been removed from develop branch, so make sure this dublincore folder and submodule is removed before merging when it is ready to merge.